runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -446,6 +446,11 @@ private ILogData cacheLoadAndGet(@NonNull Cache<Long, ILogData> cache, long addr
             return loadedValue;
         }
 
+        // Exclude all checkpoint entries, which might be 25MB and cause OOM
```

Comment:
You can point out that these entries don't need to be cached because they are not used when rolling forward/backwards to materialize a snapshot.